geoopt/manifolds/sphere.py

```diff
@@ -97,6 +97,10 @@ def _dist(self, x, y):
         inner = self._inner(None, x, y).clamp(-1, 1)
         return torch.acos(inner)
 
+    def rand_(self, x):
+        x.set_(self._projx(x))
```

Comment:
That seems to be OK to implement all inplace sampling operations like this, btw. They are not frequently used and memory footprint is not that much.